docs/md/modules-covid.md

```diff
@@ -0,0 +1,166 @@
+# Covid namespace
```

Comment:
This file needs to be added to the left navigation bar under "Module-Specific Usage".

https://github.com/broadinstitute/wfl/blob/develop/docs/mkdocs.yml#L38-L43

Reply:
(And, nit: I'd stylize this as COVID namespace myself.)